Sources/ComposableArchitecture/SwiftUI/ActionSheet.swift

```diff
@@ -118,18 +118,41 @@ public struct ActionSheetState<Action> {
   public var buttons: [Button]
   public var message: TextState?
   public var title: TextState
+  public var titleVisibility: Visibility
 
   public init(
     title: TextState,
+    titleVisibility: Visibility = .automatic,
```

Comment:
should we do another `init` that doesn't take `titleVisibility` and mark its availability?